### ISSUE TITLE
When i verify my email, the accounts get locked with: Something's Going on Here, and i am getting asked to verify my phone

### ISSUE THREAD

**Issue by Viktigboy** [NightfallGT/Discord-Account-Generator#43]
help please :)

**Comment by csenjoyer1337** [comment]
Your IP / proxy is blacklisted, you can get free proxy at webshare.io which worked for me ,but your tokens wont last long no matter what.

**Comment by Viktigboy** [comment]
That makes sense because I used a free proxy.

I tried without, and it sill locked me out.

I also tried creating an account with a mail that's not temporary, and it worked.

**Comment by csenjoyer1337** [comment]
> That makes sense because I used a free proxy.
> 
> I tried without, and it sill locked me out.
> 
> I also tried creating an account with a mail that's not temporary, and it worked.

yeah dont matter it now, owner says its patched atm, but i cant seem to find working one yet

**Comment by Viktigboy** [comment]
Thank you for the support.
Imma start using it again

**Comment by arin2115** [comment]
But, the problem is that, Discord detects ChromeDriver, not the mail or something. Only fix to auto phone verification is waiting for new undetected driver.
